input/cql/HIVConcepts.cql

```diff
@@ -178,8 +312,8 @@ code "Facility-level testing": 'HIV.B.DE17' from "HIVConcepts" display 'Facility
 code "Currently pregnant - HIV.B.DE29": 'HIV.B.DE29' from "HIVConcepts" display 'Currently pregnant'
 code "Breastfeeding - HIV.B.DE32": 'HIV.B.DE32' from "HIVConcepts" display 'Breastfeeding'
 code "Partner HIV status reported": 'HIV.B.DE33' from "HIVConcepts" display 'Partner HIV status reported'
-code "HIV-positive - HIV.B.DE34": 'HIV.B.DE34' from "HIVConcepts" display 'HIV-positive'
-code "HIV-negative - HIV.B.DE35": 'HIV.B.DE35' from "HIVConcepts" display 'HIV-negative'
+code "HIV-positive": 'HIV.B.DE34' from "HIVConcepts" display 'HIV-positive'
+code "HIV-negative": 'HIV.B.DE35' from "HIVConcepts" display 'HIV-negative'
```

Comment:
Didn't we have two HIV-positive, i.e., one in HIV.B and one in HIV.D?

Reply:
Hmmm... looking at it (and this may be hard to do in a principled way) `HIV.B.DE33` (the question to which these are the implied answers) concerns partner HIV status whereas `HIV.B.DE115` concerns the client's HIV status. Of the two, it's probably less confusing if `HIV.B.DE116` and `HIV.B.DE117` are the plain "HIV-positive" and "HIV-negative" concepts.